src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,33 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
```

Comment:
It would be better to move these styles to separate file or `main.scss`, since them are not related to the `calendar` block